acceptance/auth/bundle_and_profile/script.prepare

```diff
@@ -9,8 +9,4 @@ host = https://test@bar.com
 token = test" >> "$DB_CFG"
 
 # Replace placeholder with an actual host URL
-if [[ "$OSTYPE" == "darwin"* ]]; then
-    sed -i '' -e 's|$DATABRICKS_HOST|'$DATABRICKS_HOST'|g' databricks.yml
-else
-    sed -i -e 's|$DATABRICKS_HOST|'$DATABRICKS_HOST'|g' databricks.yml
-fi
+envsubst < databricks.yml > out.yml && mv out.yml databricks.yml
```

Comment:
Thanks. You can also move databricks.yml content out of bash script and into databricks.yml.tmpl

Also, script.prepare is intended to be shared across different tests. In the same test you should just put envsubst at the top of you script.

Reply:
> You can also move databricks.yml content out of bash script and into databricks.yml.tmpl

Not sure I understood, there is no databricks.yml content in this bash script, can you elaborate?

Reply:
oh sorry I meant .databrickscfg.